### PR TITLE
[10.0][FIX]PYLINT mainly

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
After a few merges branch 10.0 is red

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
@sbidoul Can you please add a readme to the mis_builder_operating_unit module? Or if you don't mind me to create it I can add it. Thank you

**Comment by sbidoul** [comment]
I don't mind at all if you do it :)

File: account_operating_unit/tests/test_payment_operating_unit.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # © 2016 Serpent Consulting Services Pvt. Ltd.
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
 from odoo.addons.account_operating_unit.tests import\
-    test_account_operating_unit as test_ou
+    test_account_operating_unit as test_ou   # noqa
```
Comment (sbidoul) [inline account_operating_unit/tests/test_payment_operating_unit.py]: Is it complaining about indentation? If yes using parenthesis is nicer than the backslash

```python
from odoo.addons.account_operating_unit.tests import (
    test_account_operating_unit as test_ou
)
```

Comment (AaronHForgeFlow) [reply account_operating_unit/tests/test_payment_operating_unit.py]: Actually it is complaining about the relative import. It is asking for an absolute import like `from . import ...`

Anyway i'll put the brackets it's nicer. Thanks

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**